src/packageurl/contrib/django/models.py

```diff
@@ -24,53 +24,33 @@
 # Visit https://github.com/package-url/packageurl-python for support and
 # download.
 
+import warnings
+
 from django.core.exceptions import ValidationError
 from django.db import models
 from django.utils.translation import gettext_lazy as _
 
 from packageurl import PackageURL
+from packageurl.contrib.django.utils import purl_to_lookups
 
-
-def without_empty_values(input_dict):
-    """
-    Return a new dict not including empty value entries from `input_dict`.
-
-    `None`, empty string, empty list, and empty dict/set are cleaned.
-    `0` and `False` values are kept.
-    """
-    empty_values = ([], (), {}, "", None)
-
-    return {key: value for key, value in input_dict.items() if value not in empty_values}
-
-
-def purl_to_lookups(purl):
-    """
-    Return a lookups dict built from the provided `purl` string.
-    Those lookups can be used as QuerySet filters.
-    """
-    if not purl.startswith("pkg:"):
-        purl = "pkg:" + purl
-
-    try:
-        package_url = PackageURL.from_string(purl)
-    except ValueError:
-        return  # Not a valid PackageURL
-
-    package_url_dict = package_url.to_dict(encode=True)
-    return without_empty_values(package_url_dict)
+warnings.warn(
+    "`purl_to_lookups`, `without_empty_values` is moved from packageurl.contrib.django.models"
+    " to packageurl.contrib.django.utils",
+    DeprecationWarning,
+)
```

Comment:
You need to ensure that current imports of without_empty_values and purl_to_lookups from the models module are still working for backward compatibility.
Also, you should only warn when importing from `models`, you do not want to warn when loading this module.

Reply:
Thanks! Made the requested changes